models/models.go

```diff
@@ -26,6 +26,16 @@ const (
 // StepListItemModel ...
 type StepListItemModel map[string]stepmanModels.StepModel
 
+// PipelineModel ...
+type PipelineModel struct {
+	Stages []string `json:"stages,omitempty" yaml:"stages,omitempty"`
```

Comment:
How would we extend this if we wanted to enable additional info for a given stage (e.g env vars)?

Reply:
We agreed to go with a string only solution for now and we investigate if there's a valid use case for making these more configurable.